web-app/package.json

```diff
@@ -1,45 +1,45 @@
 {
-  "name": "web-app",
-  "version": "0.0.0",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build",
-    "test": "ng test --sourceMap=false --browsers=ChromeHeadless --watch=false",
+    "build": "ng build --prod",
+    "test": "ng test --browsers=ChromeHeadless --watch=false --code-coverage",
```

Comment:
Code coverage should be serperate ticket with the intention to implement a "tag" in the readme